### DISCUSSION TITLE
Getting error while installing chainlink {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";

### DISCUSSION THREAD

**Discussion by meetjn** [Cyfrin/foundry-full-course-cu#2916] [Q&A]
![Screenshot 2024-11-24 at 8 40 20 PM](https://github.com/user-attachments/assets/2ff63d4c-a375-4665-9099-617156e3781d)

**Comment by LordAdhaar** [comment]
It could be due to unstable internet connection. Did you run the command again or trying from a different internet connection?

**Comment by EngrPips** [comment]
> ![Screenshot 2024-11-24 at 8 40 20 PM](https://private-user-images.githubusercontent.com/141674944/389283713-2ff63d4c-a375-4665-9099-617156e3781d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI0Njk0MTQsIm5iZiI6MTczMjQ2OTExNCwicGF0aCI6Ii8xNDE2NzQ5NDQvMzg5MjgzNzEzLTJmZjYzZDRjLWEzNzUtNDY2NS05MDk5LTYxNzE1NmUzNzgxZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyNFQxNzI1MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04OTAzOTNlYTY3YjdlNWM4MjUwY2Q0MWRjNDAxZWJlMzVhMjQzYjBkZWYzNTE0N2E2MTgwODYzM2UwZmI2MTg1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.KePzCNrapdNW7F60GSy055P_ii-BPfMz3Hkw9LL-HkU)

Retry with a stable internet connection and it should work.

**Comment by meetjn** [comment]
Thanks it worked

**Reply by EngrPips** [reply]
Awesome.
